src/cmd-buildextend-metal

```diff
@@ -8,19 +8,16 @@ dn=$(dirname "$0")
 print_help() {
     cat 1>&2 <<'EOF'
 Usage: coreos-assembler buildextend-metal --help
-       coreos-assembler buildextend-metal [--build ID] [--bios] [--uefi]
+       coreos-assembler buildextend-metal [--build ID]
 
-  Build raw metal images for bios or metal. If neither switches are provided,
-  both targets are built.
+  Build raw metal images.
```

Comment:
I guess we only have one metal image now, right? So `raw metal image` ?

Reply:
`Build a bare metal image`? raw seems redundant.